analyze_logs.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+import fileinput # For accessing stdin
+
+import slack_sdk
+
+import slack_reporter_config as config
+
+
+def report_no_output():
+    client = slack_sdk.WebClient(token=config.auth_token)
+    text = ("The canary tests on the {} board had no recent output!"
```

Comment:
not sure if you were aware, but you can use direct string interpolation for things like this, 
ex.
```
test = f"The canary tests on the {config.board_name} board had not recent output!"
```

Reply:
That's a good reminder; I still haven't gotten used to format strings. Thanks!